stories/Components/CookieConsentBanner/CookieConsentBanner.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect } from 'react';
+import 'vanilla-cookieconsent/dist/cookieconsent.css';
+import * as CookieConsent from 'vanilla-cookieconsent';
+// Make the imported module available globally for storybook reset and other direct calls if needed
+if (typeof window !== 'undefined') {
+  window.CookieConsent = CookieConsent;
+}
+
+const defaultConfig = {
```

Comment:
Can we break this out as a standalone `cookieconsent-config.js` that the JSX imports? That way, if an application wants to use the `<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/orestbida/cookieconsent@3.1.0/dist/cookieconsent.css">` approach, they can just reference the config.js file.